internal/otelcontribcore/main.go

```diff
@@ -22,7 +22,7 @@ import (
 	"log"
 
 	"go.opentelemetry.io/collector/component"
-	"go.opentelemetry.io/collector/service"
+	"go.opentelemetry.io/collector/otelcol"
```

Comment:
Was this a deprecation change?

Reply:
Yes, still working through more of the changes